docs/hooks/use-resolved-path.md

```diff
@@ -22,7 +22,7 @@ This is useful when building links from relative values and used internally for
 
 ## Splat Paths
 
-The original logic for React Router's `useResolvedPath` hook behaved differently for splat paths which in hindsight was incorrect/buggy behavior. Please see the [React Router Docs][rr-use-resolved-path-splat] for a longer explanation but this was determined to be a "breaking bug fix" and was fixed behind a future flag in React Router and exposed up through a [`v3_relativeSplatPath`][remix-config-future] Future Flag in Remix. This will become the default behavior in Remix v3, so it is recommended to update your applications at your convenience to be better prepared for the eventual v3 upgrade.
+The original logic for React Router's `useResolvedPath` hook behaved differently for splat paths that in hindsight were incorrect/buggy behavior. Please see the [React Router Docs][rr-use-resolved-path-splat] for a longer explanation, but this was determined to be a "breaking bug fix" and was fixed behind a future flag in React Router and exposed up through a [`v3_relativeSplatPath`][remix-config-future] Future Flag in Remix. This will become the default behavior in Remix v3, so it is recommended to update your applications at your convenience to be better prepared for the eventual v3 upgrade.
```

Comment:
```suggestion
The original logic for React Router's `useResolvedPath` hook behaved differently for splat paths that in hindsight was incorrect/buggy behavior. Please see the [React Router Docs][rr-use-resolved-path-splat] for a longer explanation, but this was determined to be a "breaking bug fix" and was fixed behind a future flag in React Router and exposed up through a [`v3_relativeSplatPath`][remix-config-future] Future Flag in Remix. This will become the default behavior in Remix v3, so it is recommended to update your applications at your convenience to be better prepared for the eventual v3 upgrade.
```